README.md

```diff
@@ -36,7 +56,7 @@ While we take pains to keep our documentation updated, the best source of inform
 Getting set up
 --------------
 
-It's easy to get set up: just some pieces of software to install and in you'll be running the tests! 
+It's easy to get set up: just some pieces of software to install and in you'll be running the tests!
```

Comment:
I think there is an error in this sentence. The part that reads `and in you'll be` seems like it's missing a word. I realize this isn't part of the change made in this PR, but as long as the file is being updated, this problem should be fixed.